Source/ARTRealtime.m

```diff
@@ -1581,4 +1583,25 @@ - (ARTLocalDevice *)device {
 }
 #endif
 
+#pragma mark - Utils
+
++ (double)backoffCoefficientWithNumber:(NSInteger)number {
+    double coeff = MIN((NSTimeInterval)(number + 2.0) / 3.0, 2.0);
+    return coeff;
+}
+
++ (double)jitterDelta {
+    return 0.2;
+}
+
++ (double)jitterCoefficient {
+    double coeff = 1.0 - 0.1 * arc4random_uniform(10) * self.jitterDelta;
+    return coeff;
+}
+
++ (NSTimeInterval)retryDelayFromTimeout:(NSTimeInterval)timeout retryCount:(NSInteger)retryCount {
```

Comment:
I find `retryCount` to be a bit unclear – does it mean the number of retries that have already occurred (in which case we are now on the `retryCount + 1`<sup>th</sup> retry) or the number of the current retry?

Also, instead of just "timeout", how about using the language "initial retry timeout" from RTB1?

```suggestion
+ (NSTimeInterval)retryDelayFromInitialRetryTimeout:(NSTimeInterval)initialRetryTimeout forRetryNumber:(NSInteger)retryNumber {
```

Reply:
https://github.com/ably/ably-cocoa/pull/1520/commits/171efb801fb805bb3d47064c713221affaf6405a

Reply:
>retryCount to be a bit unclear

https://github.com/ably/ably-cocoa/pull/1520/commits/aed866f7ecfb7e1ed9fa4841dea5845337158d89